dynamiq/nodes/llms/base.py

```diff
@@ -51,6 +51,9 @@ class BaseLLM(ConnectionNode):
         presence_penalty (float | None): Penalize new tokens based on their existence in the text.
         frequency_penalty (float | None): Penalize new tokens based on their frequency in the text.
         tool_choice (str | None): Value to control which function is called by the model.
+        inference_mode (InferenceMode): Mode of inference. Defaults to InferenceMode.DEFAULT.
+        schema_ (Dict[str, Any]): schema_ for structured output or function calling. Defaults to empty dict.
```

Comment:
why schema_ with underscore?

Reply:
We encountered an error with Pydantic validation related to the 'schema' attribute due to Pydantic's internal methods. Therefore, I have decided to use aliases instead.

Reply:
`schema` is a deprecated function, but it still exists. So we defined `schema_` but with `schema` alias. Fo for initialization/json manipulation and so on, it looks like `schema`.